tasks/opensuse/install.yml

```
@@ -0,0 +1,18 @@
+---
+- name: openSUE | Install Repo
```

```suggestion
- name: OpenSUSE | Install Repo
```

Update these to satisfy the linting case expectation

---

https://github.com/Jamdoog/ansible-role-tailscale/commit/ab9305f3eeba8dc3a02e64f3867141ecc431523a